apps/studio/components/interfaces/Database/Replication/DestinationPanel.tsx

```diff
@@ -118,22 +127,25 @@ const DestinationPanel = ({
       // For now, the password will always be set as empty for security reasons.
       serviceAccountKey: destinationData?.config?.big_query?.service_account_key ?? '',
       publicationName: pipelineData?.config.publication_name ?? '',
-      maxSize: pipelineData?.config?.batch?.max_size ?? 1000,
-      maxFillMs: pipelineData?.config?.batch?.max_fill_ms ?? 10,
-      maxStalenessMins: destinationData?.config?.big_query?.max_staleness_mins ?? 5,
-      enabled: existingDestination?.enabled ?? true,
+      maxSize: pipelineData?.config?.batch?.max_size,
+      maxFillMs: pipelineData?.config?.batch?.max_fill_ms,
+      maxStalenessMins: destinationData?.config?.big_query?.max_staleness_mins,
     }),
-    [destinationData, pipelineData, existingDestination]
+    [destinationData, pipelineData]
   )
+
   const form = useForm<z.infer<typeof FormSchema>>({
     mode: 'onBlur',
     reValidateMode: 'onBlur',
     resolver: zodResolver(FormSchema),
     defaultValues,
   })
+
   const onSubmit = async (data: z.infer<typeof FormSchema>) => {
```

Comment:
There's quite a bit of duplicated logic between this and apply and run - if we can consolidate a bit